ui/packages/catalog-ui-search/src/main/webapp/component/property/property.hbs

```diff
@@ -23,7 +23,7 @@
         <span class="if-editing fa fa-undo property-revert is-button"></span>
         <span class="property-validation fa fa-exclamation-triangle is-hidden"></span>
     </div>
-    <label for="{{cid}}">
+    <label for="{{cid}}" title="{{label}}{{#unless label}}{{id}}{{/unless}}">
```

Comment:
❓ So the title falls back to `id` if `label` is falsey?

Reply:
Yup.

Reply:
✏️ I understand the semantic equivalency, but is there any reason not to use the clearer `{{if label}}{{label}}{{else}}{{id}}{{/if}}` instead?

Reply:
Just trying to keep you guys on your toes 😉  I'll switch it over.